src/screens/charging-stations/connector-details/ChargingStationConnectorDetails.tsx

```diff
@@ -232,24 +232,22 @@ export default class ChargingStationConnectorDetails extends BaseAutoRefreshScre
     const startStopTransactionButtonStatus = this.getStartStopTransactionButtonStatus(connector);
     // Compute Duration
     const durationInfos = this.getDurationInfos(transaction, connector);
-    // Get the provider
-    const securityProvider = this.centralServerProvider.getSecurityProvider();
     // Set
     this.setState({
       chargingStation,
       connector: chargingStation ? Utils.getConnectorFromID(chargingStation, connectorID) : null,
       transaction,
       siteImage: siteImage ? siteImage : this.state.siteImage,
       userImage: userImage ? userImage : transaction ? this.state.userImage : null,
-      isAdmin: securityProvider ? securityProvider.isAdmin() : false,
+      isAdmin: this.securityProvider ? this.securityProvider.isAdmin() : false,
```

Comment:
FYI, we are in the process to not rely on roles but only on authorizations.
Soon you will have to get rid of existing isAdmin, isXXX and replace them by canXXX